templates/vue3/vue.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    chainWebpack: config => {
+        config
+            .plugin('html')
+            .tap(args => {
+                args[0].title = "My NEAR Vue App";
```

Comment:
should this also use `near-blank-project` and be added to that `replaceInFiles` block in `index.js`?

Reply:
You're right, I'll do that.

Reply:
Found a better way during App creation. I'll remove the vue.config.js